content/en-us/players/tools.md

```diff
@@ -169,12 +169,14 @@ The following example shows steps for adding a `Class.Script` on the server that
 2. Copy the following code and paste it into your `Class.Script`.
 
    ```lua
+   local Lighting = game:GetService("Lighting")
+
    local tool = script.Parent
    local function onActivate()
-    if game.Lighting.ClockTime \>= 8 and game.Lighting.ClockTime < 16 then
-      game.Lighting.ClockTime = 20
+    if Lighting.ClockTime \>= 8 and Lighting.ClockTime < 16 then
```

Comment:
Stray backslash invalidates the first operator (\>= is not a valid).

*Note: this is out of scope of the PR, as this is not related to changing dot indexing services to use GetService. I've therefore left this as a comment and not a request for change.*

Reply:
I sent a PR to change that earlier but forgot to change it back when the PR was merged, thanks for notifying me.